packages/algob/test/internal/cli/project-creation.ts

```diff
@@ -0,0 +1,60 @@
+import { assert } from "chai";
+import fs from "fs-extra";
+
+import { ERRORS } from "../../../src";
+import { createProject } from "../../../src/internal/cli/project-creation";
+import { expectBuilderErrorAsync } from "../../helpers/errors";
+import { useFixtureProject } from "../../helpers/project";
+
+describe("Init project", () => {
+  useFixtureProject('init-task');
+
+  afterEach(() => {
+    const paths = fs.readdirSync("./");
+    for (const path of paths) {
+      if (path !== "README.md") { fs.removeSync(path); }
+    }
+  });
+
+  it("should init project in a empty folder(javascript)", async () => {
+    const location = "test-project";
+    await createProject(location, false);
+
+    assert.isTrue(fs.existsSync(`./${location}/algob.config.js`));
+    assert.isTrue(fs.existsSync(`./${location}/scripts/0-sampleScript.js`));
+  });
+
+  it("should init project in a empty folder(typescript)", async () => {
+    const location = "test-project";
+    await createProject(location, true);
+
+    assert.isTrue(fs.existsSync(`./${location}/algob.config.js`));
+    assert.isTrue(fs.existsSync(`./${location}/scripts/0-sampleScript.ts`));
```

Comment:
Please also add an assertion for `tsconfig.json` (since we're initializing a typescript project)

Reply:
ok